source/release-notes.txt

```diff
@@ -10,6 +10,33 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.42.3
+----------------
+
+*Released March, 20 2024*
+
+New Features
+
+- Install updates without confirmation in the background by default
+  (:issue:`COMPASS-7616`)
+- Disable search indexes via preferences (:issue:`COMPASS-7595`)
+- Setup provider to share current connection info (:issue:`COMPASS-7701`)
```

Comment:
We probably don't need to include this in our Release notes as this is a change that has no user facing effect. I would say the asme for "Bump OIDC dependencies to latest versions". @gribnoysup can you please confirm?

Reply:
Yes, shouldn't be here. A bunch of stuff here is not user facing for Compass desktop users and should not be here, seems like we should pay better attention during code reviews

Reply:
Ok so I just updated the release notes on our Github release and to summarise, we can remove:
- Setup provider to share current connection info (:issue:`COMPASS-7701`)
- Show performance advisor banner (:issue:`COMPASS-7596`)
The rest can stay. Thank you :)